Dockerfile_maven

```diff
@@ -0,0 +1,99 @@
+# syntax = docker/dockerfile:1.0-experimental
```

Comment:
Why do you need another Dockerfile for this (Dockerfile_maven)?
Isn't the idea to replace checking out and building the plugins with just getting them via maven?